doc/lua_api.txt

```diff
@@ -1964,6 +1964,23 @@ Elements
 * End of a container, following elements are no longer relative to this
   container.
 
+### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<direction>;<multiplier>]`
+
+* Start of a scroll_container block, all elements in a scroll_container block are
+  * moved by `(X, Y),
+  * additionally moved by the current value of the scrollbar with the name
+    `scrollbar name` times `multiplier` in the direction `direction` and
+  * clipped to the rectangle defined by `X`, `Y`, `W` and `H`.
```

Comment:
Not sure if this is even needed. Textareas clip too, since it's what you'd expect from scrollbar-containers.

Reply:
Normal containers do not clip. So I think, it makes sense pointing this out.